frontend/store/auctions.ts

```diff
@@ -22,7 +21,7 @@ let updateAuctionsPricesIntervalId: ReturnType<typeof setInterval> | undefined;
 
 interface State {
     auctionStorage: Record<string, AuctionTransaction>;
-    eventStorage: Record<string, Event[]>;
+    takeEventStorage: Record<string, Event[]>;
```

Comment:
What is the `Event` type, where did you get it? I suggest that you define our own type and store only things we need, ie transactionHash at the moment (and maybe a block number)

Reply:
I get it directly from `ether.js`. 
https://docs.ethers.io/v5/api/contract/contract/#Contract-queryFilter

It is also the type that `contract.queryFilter` returns, which is why I would continue using it.

Reply:
As suggested, we may not want to put the whole `Event` object into the store 🙂

Reply:
I made a smaller version of the object!